pytim/__init__.py

```diff
@@ -306,9 +306,14 @@ def assign_universe(self, universe, radii_dict=None, warnings=False):
             self.interface.warnings = warnings
             if radii_dict is not None:
                 self.interface.radii_dict.update(radii_dict)
+            if isinstance(universe, MDAnalysis.core.universe.Universe):
```

Comment:
the checks should be moved before accessing `universe` at line 304. 
I would suggest something like:

        def assign_universe(self, input, radii_dict=None, warnings=False):
            try:
                if isinstance(input, MDAnalysis.core.universe.Universe):
                    self.interface.universe = input
                elif isinstance(input, MDAnalysis.core.groups.AtomGroup):
                    self.interface.universe = input.universe

                self.interface.all_atoms = self.universe.select_atoms('all')
                self.interface.radii_dict = tables.vdwradii.copy()
                self.interface.warnings = warnings
                if radii_dict is not None:
                    self.interface.radii_dict.update(radii_dict)
            except BaseException:
                raise Exception(self.interface.WRONG_UNIVERSE) 
            self._missing_attributes(self.interface.universe)

where i changed `universe` -> `input` just to avoid confusion  

See also the comment below about the `itim_group`

Reply:
Done. However, do not use input as a name, it's a built-in. I renamed it input_obj.

Reply:
can't see the changes, did you do a push -f ?  In case please close this PR and open a new one.